src/libs/actions/Session.js

```diff
@@ -208,19 +208,21 @@ function restartSignin() {
  *
  * @param {String} password
  * @param {String} validateCode
+ * @param {String} [accountID]
+ * @param {String} [email]
  */
-function setPassword(password, validateCode) {
+function setPassword(password, validateCode, accountID, email) {
```

Comment:
I figured it would be good to leave in email since the API endpoint takes in either email or accountID. May be good for future use, but if it isn't going to be used anywhere else it's probably better just to get rid of it!